runtime/integration-tests/src/generic/cases/liquidity_pools.rs

```diff
@@ -1178,6 +1185,62 @@ mod centrifuge {
 			));
 		}
 
+		pub fn register_lp_eth_usdc<T: Runtime>() {
```

Comment:
I think there's another better way of dealing with this, as Luis noted in a previous PR - https://github.com/centrifuge/centrifuge-chain/blob/main/runtime/integration-tests/src/generic/utils/currency.rs

Reply:
Not sure how this will work. There is no location that I need. Can you be more precise